iotedgedev/deploymentmanifest.py

```diff
@@ -147,3 +186,74 @@ def _get_module_content(self):
             return self.json["moduleContent"]
         else:
             raise KeyError("modulesContent")
+
+    def _validate_deployment_manifest_schema(self):
```

Comment:
the method has lots of duplication with validate_deployment_template. Can we refactor?

Reply:
Refactored